mne/io/reference.py

```diff
@@ -193,11 +200,19 @@ def add_reference_channels(inst, ref_channels, copy=True):
                      'unit': FIFF.FIFF_UNIT_V,
                      'coord_frame': FIFF.FIFFV_COORD_HEAD,
                      'loc': np.zeros(12)}
+        # see if "zeroth" EEG electrode present in dig points
+        dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+        if len(dig_loc) == 1:  # is either 0 or 1, never >1 for Neuromag
+            chan_info['loc'][:3] = dig_loc[0]['r'][:]
+            chan_info['loc'][3:6] = dig_loc[0]['r'][:]  # 3:6 holds ref loc
```

Comment:
Could you add a check for `if len(ref_channels) > 1`? When that is the case, the dig_loc should not be used. Otherwise, the two reference channels will have the same location, which breaks topo plots. In this case, it would be better to raise a warning and not use any location information.